ciphey/basemods/Decoders/atbash.py

```diff
@@ -0,0 +1,50 @@
+# atbash.py
```

Comment:
The name of this file isn't needed as a comment, as the file name is atbash.py :)

Can you change this to:

```
# community
```

to denote this is a community module? will be useful for tagging system #147 :sparkles: